### PR TITLE
Implement review page design

### PR DESCRIPTION
**Pull request by nbuonin**
See #34 first

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -5,46 +5,74 @@
         {{ partial "skip.html" . }}
         {{ partial "nav.html" . }}
         <main id="content">
+            <div class="container p-0">
+                <div id="hp-hero" class="jumbotron-fluid">
+                    <h1 class="display-4">Help your patients who are at risk for opioid addiction.</h1>
```
Comment (sdreher) [inline layouts/index.html]: The text overlaying the image + button is hard to read and a somewhat dodgy shade of brown.

File: layouts/index.html [hunk 2]
```diff
@@ -5,46 +5,74 @@
         {{ partial "skip.html" . }}
         {{ partial "nav.html" . }}
```
Comment (sdreher) [inline layouts/index.html]: On the homepage, consider pulling the nav. it's not really needed on this page and will distract from the rest of the content on the page.

File: static/scss/base/vertical-nav.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
+$predecessor: rgb(137, 179, 157);
+$active: rgb(230, 178, 87);
+
+nav[aria-labelledby="knowledge-domain-navigation"] {
```
Comment (sdreher) [inline static/scss/base/vertical-nav.scss]: Hmmmm....these styles are quite rich, but can be distracting from the main content. Let's review together on Monday.

Comment (nbuonin) [reply static/scss/base/vertical-nav.scss]: Yeah, agreed. Thanks for looking, and lets review Mon.

**Pull request merged**